### ISSUE TITLE
Refrigerator - Inoperative

### ISSUE THREAD

**Issue by pcbaston** [CanberraOceanRacingClub/namadgi3#701]
Sam reported:
"Not sure if anyone else has experienced the Bar fridge going off line,  with power light blinking on the control panel. 
I think it's normal when we first turn the fridge in that the light blinks for about 15 min. 
I have had the blinking light happen in the past, but the fridge resets and starts up after a short, 15 to 30 minutes" 
and
"The fridge (see spec pages) has been **off line now for 24 hours** and upon investigation, it seems the control circuit is faulty.
Dimitri and I pulled the fridge out and when we disconnected the power to the controller, the circuit on the boat control panel stopped flashing and indicator remained on. Not flashing.
I will see if there is a techy in Vuda Marina that can have a look to try and fix, but I am not confident given the limitations in Fiji.
My call would be to see if we can order a new controller (see pic) and I can bring it with me on 9 July.
Any comments or thoughts?

Controller:
![pro-j6lxRgvX](https://github.com/CanberraOceanRacingClub/namadgi3/assets/120452016/0f954a02-96d8-41dc-8808-55be76224dbc)

Refrigerator is a CoolMate CRX- series refrigerator - manual:
[coolmatic_crx50.pdf](https://github.com/CanberraOceanRacingClub/namadgi3/files/11859697/coolmatic_crx50.pdf)

**Comment by pcbaston** [comment]
Michael Martin responded:
"On occasion the fridge has done the same thing to me – started flashing for no apparent reason.
I usually fix it by turning it off on the switchboard waiting a few minutes and turning it back on again.
This is not a new issue unless something has changed."

**Comment by pcbaston** [comment]
I emailed Sam:
"just looking at the manuals I can find on the fridge & controller (apparently they are not supposed to be working together!!).
Anyway, an easy test would be to connect the controller to another 12VDC power source on the control panel with 10 to 15 Amps protection.
I need to look at the control panel manual, but any fault with the fridge should be indicated by a LED inside the fridge (Green for ON and RED flashing for fault).  It would be helpful if you could confirm any fault indication of the fridge first, then work backwards.

There is a repair agent in Sydney for the controller - or a WANCOOL replacement can be sourced [here](https://www.everythingcaravans.com.au/products/control-unit-wancool) for about $255 plus postage - but I'd want to be sure its the controller!  Seems they are a pretty common component, so you may be able to source one in Fiji."

**Comment by pcbaston** [comment]
Found the Dometic manual for the CFX series compressor refrigerators.
[dometic-coolmatic-crx-65-e_9600028417_82511.pdf](https://github.com/CanberraOceanRacingClub/namadgi3/files/11859732/dometic-coolmatic-crx-65-e_9600028417_82511.pdf)

**Comment by pcbaston** [comment]
@peterottesen commented:
" We have had this problem for a few years now. I just turn the switch on and off, repeating this cycle, until it turns on and there is a steady light. It does not take long. "

**Comment by pcbaston** [comment]
@delcosta updated:
"After 48 hours, the bar fridge on Namadgi self-revived. I am not sure what we did. However, we had a windlass failure which ended up being a loose contact at the anchor control fuse.
Upon reuniting contacts the fridge was noticed to be on again and so far ok.
The lack of ice was painful and we resorted to trading fresh fish for ice with another yacht.
So for now that item doesn't need action.
Just the door bracket and latch."

Issue CLOSED.
